app/views/job_templates/_job_template_callback_tab_content.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="tab-pane" id="ansible_callback_enabled">
+  <%= checkbox_f f, :ansible_callback_enabled, :label=>_('Enable Ansible Callback'), :disabled => @template.locked? %>
```

Comment:
It is somewhat unfortunate that right now the tab is displayed even if the provider type is not ansible. I was able to put together a makeshift solution with jquery, although I don't really dare to suggest you to go down that path.

@MariaAga If we have an old erb page that we're extending from a plugin using pagelets and we want the content to dynamically change based on field from the original, non-extended page, do we have any other option apart from registering an onchange handler onto the fields we're interested in?

Reply:
pagelets are really not my area, so we might have to use that suggested jquery